src/message.coffee

```diff
@@ -54,8 +60,11 @@ class LeaveMessage extends Message
 # text - A String of the new topic
 # id   - A String of the message ID.
 class TopicMessage extends TextMessage
+  type: 'topic'
 
 class CatchAllMessage extends Message
+  type: 'all'
```

Comment:
This should probably be 'catchall' or something a little more specific than 'all'